csrc/device_lower/pass/predicate.cpp

```diff
@@ -204,6 +275,11 @@ class ConditionalFromPredicateModifier : public kir::ExprMutator {
 
   // Keep track of the loop in which the currently visiting expr is a rotated.
   std::unordered_set<ForLoop*> rotated_loop_;
+
+  kir::IfThenElse* current_elect_sync_ = nullptr;
```

Comment:
Add comments for these member variables.

Reply:
added 
```
  // Keep track of the ite whose predicate is ElectSync to identify the
  // TMA load branch.
  kir::IfThenElse* current_elect_sync_ = nullptr;

  // Ublk predicate value is specialized to hoist to ElectSync to predicate
  // both TMA load and MBarrierArriveExpectTx.
  Val* ublk_predicate_val_ = nullptr;

  // The Ublk predicate generated in the TMA load branch needs to be applied
  // in the computation branch to predicate MBarrierWaitParity. Although both
  // for-loops share the same iteration domain, their loop indices differ.
  // Here, we record the index from the TMA load branch to later replace it
  // with the corresponding index from the computation branch.
  Val* tma_branch_fl_index_ = nullptr;
```